src/popup/router/pages/About.vue

```diff
@@ -1,76 +1,190 @@
 <template>
   <div class="about">
-    <Logo class="logo" />
-    <p>
-      <span class="version">
-        commit
-        <a
-          :href="`${COMMIT_URL}${commitHash}`"
-          target="_blank"
-        >
-          {{ commitHash.slice(0, 7) }}</a>
-        / {{ extensionVersion }}
-      </span>
-    </p>
-    <hr>
-    <div class="waellet-links">
+    <div class="table">
+      <div class="table-item">
+        <div class="name">
+          {{ $t('pages.about.name') }}
+        </div>
+        <div class="value">
+          Superhero Wallet
```

Comment:
`font-weight` should be 400
![image](https://user-images.githubusercontent.com/9008074/189042197-31294d86-acbe-4457-b0d4-f734a2d72bed.png)
![image](https://user-images.githubusercontent.com/9008074/189042260-37155819-8468-44b0-ac67-26a3d7fac82d.png)

Reply:
it's due to figma font difference with 300 it looks exact as 400